### PR TITLE
chore: post some ssl info on startup

### PR DESCRIPTION
**Pull request by pajlada**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -90,6 +91,22 @@ int main(int argc, char **argv)
             attachToConsole();
         }
 
+        qCInfo(chatterinoApp)
+            << "Chatterino Qt SSL library build version:"
+            << QSslSocket::sslLibraryBuildVersionString()
+            << ", library version:" << QSslSocket::sslLibraryVersionString();
```
Comment (Nerixyz) [inline src/main.cpp:97]: This comma will float in space. This should probably use `.nospace()`.

**Review by Nerixyz — COMMENTED** [review]
I don't think the info category is appropriate here. It's printing debug information, thus it should use `qCDebug`.

File: src/main.cpp (hunk 1 above)
Comment (pajlada) [reply src/main.cpp]: fixed in [`7225b74` (#5475)](https://github.com/Chatterino/chatterino2/pull/5475/commits/7225b7475dc09ca84d89ced80a281d706a5db94e) by splitting the library build & library version strings up into separate prints

**Comment by pajlada** [comment]
keeping it as info

**Pull request merged**